js/base-util.js

```diff
@@ -105,52 +100,181 @@ function baseUtil () {
 		`, true);
 	};
 
-	d20plus.ut.chatTag = (message) => {
-		const isStreamer = !!d20plus.cfg.get("interface", "streamerChatTag");
-		const scriptName = isStreamer ? "Script" : message;
+	d20plus.ut.chatTag = () => {
+		const legacytStyle = !!d20plus.cfg.getOrDefault("chat", "legacySystemMessagesStyle");
+		const showWelcome = !!d20plus.cfg.getOrDefault("chat", "showWelcomeMessage");
+		const isStreamer = !!d20plus.cfg.get("chat", "streamerChatTag");
+		const classname = !legacytStyle ? "userscript-commandintro userscript-b20" : "userscript-hackerintro";
+		const scriptName = isStreamer ? "Script" : d20plus.scriptName;
+		const data = [
+			d20plus.scriptName,
+			window.r20es?.hooks?.welcomeScreen?.config?.previousVersion,
+			d20plus.ut.WIKI_URL,
+		];
+		const welcomeTemplate = (b20v, vttv, faq) => `
+			<div class="${classname}">
+				<img src="" class="userscript-b20img" style="content: unset; width:40px;position: relative;top: 10px;float: right;">
+				<h1 style="display: inline-block;line-height: 25px;margin-top: 5px; font-size: 22px;">
+					betteR20 
+					<span style=" font-size: 13px ; font-weight: normal">by 5etools</span>
+					<p style="font-size: 11px;line-height: 15px;font-family: monospace;color: rgb(32, 194, 14);">${b20v} loaded<br>VTTES v${vttv} detected</p>
+				</h1>
+				<p>Need help? Visit our <a href="${faq}/index.php/BetteR20_FAQ"><strong>wiki</strong></a> or join our <a href="https://discord.gg/nGvRCDs"><strong>Discord</strong></a>.</p>
+				<span title="You'd think this would be obvious.">
+					<p>Please DO NOT post about this script or any related content in official channels, including the Roll20 forums.</p>
+					<p>Before reporting a bug on the Roll20 forums, please disable the script and check if the problem persists.</p>
+				</span>
+			</div>
+		`;
+		if (showWelcome) {
+			if (!isStreamer) {
+				d20plus.ut.sendHackerChat(welcomeTemplate(...data));
+			} else {
+				d20plus.ut.sendHackerChat(`VTTE detected and ${scriptName} successfully loaded.<br>`);
+			}
+		}
 		if (window.enhancementSuiteEnabled) {
-			d20plus.ut.sendHackerChat(`
-				VTT Enhancement Suite detected.
-				<br><br>
-				${scriptName} initialised.
-				<br>
-			`);
-		} else d20plus.ut.showHardDickMessage(scriptName);
-		d20plus.ut.sendHackerChat(`
-			${isStreamer ? "" : `
-			<br>
-			Need help? Visit our <a href="${d20plus.ut.WIKI_URL}/index.php/BetteR20_FAQ">wiki</a> or join our <a href="https://discord.gg/nGvRCDs">Discord</a>.
-			<br>
-			<br>
-			<span title="You'd think this would be obvious.">
-			Please DO NOT post about this script or any related content in official channels, including the Roll20 forums.
-			<br>
-			<br>
-			Before reporting a bug on the Roll20 forums, please disable the script and check if the problem persists.
-			</span>
-			`}
-		`);
+			$("#lamer-progress").before(`<span><span>&gt;</span>vtt enhancement suite detected</span>`)
```

Comment:
`lame` is a Gamer:tm: word these days, and I've already been told off for using `blacklist`. Switch it out for e.g. `boring-progress`

Reply:
Didn't know that. To me, lamer was a common contradistinction to a "hacker" back in the early 00s.

Anyway, boring it is